<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>-10</y>
      <width>661</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_main">
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索：</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_select">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>621</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_hotpot">
       <widget class="QListWidget" name="listWidget_hotpotarticles">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>611</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_hotpotarticles">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>601</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>热点文章</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_">
       <widget class="QLabel" name="label_serachresult">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>601</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>搜索结果</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_serachresult">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>611</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_return">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_wordAnalysis">
     <widget class="QLabel" name="label_test1_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>581</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>这是词频界面</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_stock"/>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>561</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>主页</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>词频分析</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>股票行情</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_select">
    <property name="title">
     <string>请选择来源</string>
    </property>
    <widget class="QMenu" name="menu_sina">
     <property name="title">
      <string>新浪√</string>
     </property>
     <addaction name="action_modifysina"/>
    </widget>
    <widget class="QMenu" name="menu_zhenjuan">
     <property name="title">
      <string>中国证券网√</string>
     </property>
     <addaction name="action_modifyzhenjuan"/>
    </widget>
    <addaction name="menu_sina"/>
    <addaction name="menu_zhenjuan"/>
   </widget>
   <addaction name="menu_select"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_modifysina">
   <property name="text">
    <string>禁用</string>
   </property>
  </action>
  <action name="action_modifyzhenjuan">
   <property name="text">
    <string>禁用</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
